docs/README.md

```diff
@@ -44,7 +44,7 @@ compile group: 'com.47deg', name: 'memeid', version: '@VERSION@'
 Add this to your `build.sbt` file:
 
 ```scala
-libraryDependencies += "com.47deg" % "memeid4s" % "@VERSION@"
+libraryDependencies += "com.47deg" %% "memeid4s4s" % "@VERSION@"
```

Comment:
Should be `memeid4s` instead of `memeid4s4s`

Reply:
Good catch! :) b60b0092548d75b9ac316b80d20d7363c89d3ec2